src/components/program/ProgramContentEbookReader.tsx

```diff
@@ -580,7 +577,18 @@ const ProgramContentEbookReader: React.VFC<{
   }
 
   return (
-    <div>
+    <Box height={{ base: `${!!isSafari ? '90vh' : '100vh'}`, md: '85vh' }}>
+      <CommonModal
+        isCentered
+        isOpen={modalState === 'trialComplete'}
```

Comment:
```js
<CommonModal
        isCentered
        isOpen={isTrialCompletedModalOpen}
```
Is it better?
Is modalState only one situations, right?

Reply:
I don't think `isTrialCompletedModalOpen` would be better
Because each Modal will only be open in one case, multiple Modals will not be opened at the same time.
So in the future, edit notes, delete notes, and trial completed must be merged into the same state, just like the `modalState` here.

Reply:
So you expect to combine the modals for different states into a single modal in the future, right?